### ISSUE TITLE
Meteors cant be angled

### ISSUE THREAD

**Issue by ajawsome4ev** [l2wilson94/SandPond#13]
meteors cant be angled so anything on one side of the screen cant be hit

**Comment by l2wilson94** [comment]
That is true! I wonder how we could improve it? Maybe it could fire it in the direction that the cursor last moved.
